apps/cloudfront-functions/src/__tests__/viewer-request-handler.test.ts

```diff
@@ -35,9 +35,10 @@ describe('handler', () => {
     expect(handler(makeEvent('/page')).uri).toBe('/page.html');
     expect(handler(makeEvent('/page/')).uri).toBe('/page/');
     expect(handler(makeEvent('/image.jpg')).uri).toBe('/image.jpg');
+    expect(handler(makeEvent('/font.woff2')).uri).toBe('/font.woff2'); // new test case
     const example0 = '/i-s/g/mo/v1.0/i-p/p-i/v-d';
     expect(handler(makeEvent(example0)).uri).toBe(`${example0}.html`);
     const example1 = '/i-s/g/mo/v1.0';
-    expect(handler(makeEvent(example1)).uri).toBe(`${example1}.html`);
+    expect(handler(makeEvent(example1)).uri).toBe(example1);
```

Comment:
Is this the desired behaviour?

I think it's fine, but I just wanted to gather consensus.